main/options.h

```diff
@@ -106,6 +106,7 @@ typedef struct sOptionValues {
 	bool machinable;		/* --machinable */
 	bool withListHeader;		/* --with-list-header */
 	bool interactive; /* --interactive */
+        bool secure; /* --_secure; implies --_interactive */
```

Comment:
is there is a standardized (emacs?) setting that produces your desired formatting? tabs/spaces?